l10n_ch_hr_payroll/README.rst

```diff
@@ -1,31 +1,94 @@
-Swizerland Payroll Rules.
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Switzerland Payroll Rules
 =========================
 
-**Features list :**
+This module allows you to manage the salaries of your employees
+
+** Features list :**
     * Add Swiss salary rule categories
     * Add Swiss salary rules
     * Add children in school to employee
     * Add LPP range to contract
+    * Add LPP Amount to contract.
+    * Add Worked Hours to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add Holiday Rate to contract.
 
-**For functionnal information:**
-http://ur1.ca/ir5ou
+** For further information, please visit:** http://ur1.ca/ir5ou
 
-**Author :**
-Open Net Sàrl -Industrie 59 1030 Bussigny Suisse - http://www.open-net.ch
+** Attention: **
+To prevent overwriting your salary rules changes, an update from 1.0.5 and lower to 1.0.6 and higher creates duplicates of the salary rules.
+Note that with some migrated databases, one may encounter a difficulty with the existing rules (they can not be erased if they are already used).
+The solution is then to force the existing ones to be non-updatable. 
+And this can be done using an SQL query such as the following, before upgrading to this version:
+update ir_model_data set noupdate=true where module='l10n_ch_hr_payroll' and model='hr.salary.rule' 
 
-**Contact :**
-info@open-net.ch
-
-**History :**
+Known issues / Roadmap
+======================
 
 V1.0.0: 2014-11-07/Sge
-    * Add Salary rule categories
-    * Add Salary rules
-    * Add Employee children in school
-    * Add Contract LPP rate
+    * Add Salary rule categories.
+    * Add Salary rules.
+    * Add Employee children in school.
+    * Add Contract LPP rate.
 
 V1.0.1: 2014-11-11/Sge
-    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy
+    * Set the 'LPP rate'' digits to 'Payroll Rate' decimal accuracy.
 
 V1.0.2:
     * Add some minor changes, based on pull request #66 comments.
+
+V1.0.3-4:
+    * Add LPP Amount to contract.
+    * Add Worked Hours Rate to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add new salay rules
+
+V1.0.5:
+    * Add Holiday Rate to contract.
+    * Update "Indemnité vacances 8,33%" rule to include 
+      "Holiday Rate" from contract.
+
+V1.0.6:
+    * Move salary rules from CSV file to XML file.
+    * Import salary rules only at install.
+    * Internal reorganization of files (.py and .xml).
+
+Credits
+=======
+
+.. image:: http://open-net.ch/logo.png
+   :alt: Open Net Sarl
+   :target: http://open-net.ch
+
+This module is maintained by Open Net Sarl.
+Industrie 59 
+CH-1030 Bussigny 
+
+http://www.open-net.ch
+info@open-net.ch
```

Comment:
If you're submitting this to the OCA, it means that the module is maintained collectively by the OCA. Open Net is a contributor. Please remove the paragraph.